### PR TITLE
fix(all): liveview icon color

### PR DESCRIPTION
**Pull request by m1ga**
Make the Liveview icon dark if disabled so it will fit better

### CODE REVIEW

**Comment by ewanharris** [comment]
@m1ga my concern with this change is that the colour is already used to denote whether the liveview option is disabled altogether when a packaging option is selected and I don't think it would be great to use that in a different context.

Do you find the different icons don't differentiate the state enough currently?

**Comment by m1ga** [comment]
Ah I never look at the icons this way. For me a dimmed icon/option field was associated with "inactive". So If I'm in the "package" section the "emulator" options are inactive.
If I go to "run" the :gear: icon is inactive but the for me the disabled ListView was stil perceived as "active" since it was bright. 

Often I'm in the wrong liveview state because when I look at the code and click "play" I don't really look at the :eye: icon. It would be easier to spot the current Liveview state if it bright for active and dimmed for inactive. 

Maybe we hide the icons that are not active? E.g. hide the :gear: in "run" and the :eye: in "play store"? Also the :gear: should toggle its color so you know it has two states! Should be quick style change, too.

This way we only have valid icons and both toogle the color if they are active and inactive.

**Comment by ewanharris** [comment]
I do like the idea of hiding them instead of making them disabled, then I think grey-ing out the icon when it's toggled off would fit fine and make it easier to skim over the options as you say.

I think we should keep the icon change too as not everyone works off the colour (#306)

**Comment by m1ga** [comment]
how about this:

https://user-images.githubusercontent.com/4334997/139710733-73b02f46-760f-4f95-8505-0aa05e7c57ab.mp4

**Comment by m1ga** [comment]
Still have to make some tweaks but light mode works too:

https://user-images.githubusercontent.com/4334997/139711943-b88ee677-98d1-4ff9-8cad-9c436d131572.mp4

File: styles/appc.less [hunk 1]
```diff
@@ -190,14 +194,49 @@
...
+      .icon-gear {
+        display: none;
+      }
+      .etch-octicon-eye-closed,
+      .etch-octicon-eye {
+        padding: 0 5px;
+      }
+    }
+
+    &.dist-appstore {
```
Comment (ewanharris) [inline styles/appc.less:234]: Would also need the adhoc target added

```suggestion
    &.dist-appstore, &.dist-adhoc {
```

File: lib/ui/octicon.jsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ export class Octicon {
 			classNames.push('disabled');
 		}
 
-		const fill = disabled ? '#505868' : '#9da5b4';
+		const fill = 'var(--standard-text-color)';
```
Comment (ewanharris) [inline lib/ui/octicon.jsx:67]: I'm not too sure on the change of colour here as it makes it different to everything else on the toolbar

**Review by ewanharris — COMMENTED** [review]
Clips look really good, just a couple comments

File: lib/ui/octicon.jsx (hunk 2 above)
Comment (m1ga) [reply lib/ui/octicon.jsx:67]: It is the same color the icons on the right have

Comment (ewanharris) [reply lib/ui/octicon.jsx:67]: Maybe there's a difference here on mac? For me the icon is black 
image
Whereas the other icons are grey 
image

Comment (m1ga) [reply lib/ui/octicon.jsx:67]: oh interesting. I'll check that. Perhaps its system dark/light mode that is making an issue here.

Comment (m1ga) [reply lib/ui/octicon.jsx]: looks like that variable was set by a different package and is not available on OSX.
The thing is that all buttons use `color:inherit` and get the color from the atom-panel. But there isn't a fill color. So now I use `getComputedStyle(document.querySelector("atom-panel")).color;`. 
Only "problem" it is not updated when you switch dark/light themes. You have to click it again in order to get the right color

**Review by hansemannn — APPROVED** [review]

**Pull request merged**